database/migrations/2019_12_15_0700_add_share_button_visible_option.php

```diff
@@ -21,7 +21,9 @@ public function up()
 			$table->boolean('share_button_visible')->after('downloadable')->default(false);
 		});
 
-		Album::where('id', '>', 1)
+		Album::query()
+			->withoutGlobalScopes()
+			->where('id', '>', 1)
```

Comment:
Why is `where('id', '>', 1)` needed?  :thinking: 
isn't this always true?

Reply:
I haven't spent a thought on that. I did not change that line.